### PR TITLE
fix(components): Enumeration component : change tooltip values

### PR DESCRIPTION
**Pull request by Sylvain-Talend**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Change tooltip wording for delete value and delete values

**What is the new behavior?**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://315.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://315.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://315.talend.surge.sh)

File: packages/components/stories/Enumeration.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const addItemAction = {
 };
 
 const deleteItemAction = {
-	label: 'Delete items',
+	label: 'Remove selected values',
```
Comment (mtikhinskaya) [inline packages/components/stories/Enumeration.js]: Why don't you change label for 'Delele' also?

Comment (Sylvain-Talend) [inline packages/components/stories/Enumeration.js]: You're right, it has been fixed

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://315.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://315.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://315.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://315.talend.surge.sh)

**Pull request merged**